src/studio/src/repositories/gitea-data/gitea/conf/app.ini

```
@@ -1,58 +1,44 @@
 APP_NAME = Altinn Studio Repos
-RUN_MODE = prod
 RUN_USER = git
 
 [repository]
-ROOT = /data/git/repositories
+ROOT = /var/lib/gitea/git/repositories
 
 [repository.upload]
-TEMP_PATH = /data/gitea/uploads
+TEMP_PATH = /var/lib/gitea/uploads
 
 [server]
-APP_DATA_PATH    = /data/gitea
-SSH_DOMAIN       = altinn3.no
-HTTP_PORT        = 3000
-ROOT_URL         = http://altinn3.no/repos
-STATIC_ROOT_PATH = /gitea/
-DISABLE_SSH      = true
-SSH_PORT         = 22
-DOMAIN           = altinn3.no
-LFS_START_SERVER = false
-LFS_CONTENT_PATH = /data/gitea/lfs
-LFS_JWT_SECRET   = MRlYoCcrcHmcKzUoQwaK6vKO3o4FPEJ74em5JnPRii0
-OFFLINE_MODE     = false
-
-[database]
-; Either "mysql", "postgres", "mssql" or "sqlite3", it's your choice
-DB_TYPE  = sqlite3
-; For "sqlite3" and "tidb", use absolute path when you start as service
-PATH     = /data/gitea/db/gitea.db
-HOST     = 172.18.0.2:5432
-NAME     = gitea
-USER     = root
-; NAME     = postgres
-; USER     = postgres
-PASSWD   = gitea
-SSL_MODE = disable
+APP_DATA_PATH           = /var/lib/gitea
+HTTP_PORT               = 3000
+STATIC_ROOT_PATH        = /gitea/
+; In rootless gitea container only internal ssh server is supported
+START_SSH_SERVER        = false
+DISABLE_SSH             = true
+SSH_PORT                = 22
+SSH_LISTEN_PORT         = 22
+BUILTIN_SSH_SERVER_USER = git
+LFS_START_SERVER        = false
+LFS_CONTENT_PATH        = /var/lib/gitea/git/lfs
+OFFLINE_MODE            = false
 
 [session]
-PROVIDER_CONFIG = /data/gitea/sessions
+PROVIDER_CONFIG = /var/lib/gitea/sessions
 PROVIDER        = file
-COOKIE_SECURE = false
+COOKIE_SECURE   = false
```

Default should be to enable COOKIE_SECURE
```suggestion
COOKIE_SECURE   = true
```

---

It seems to be set to true in another file https://github.com/Altinn/altinn-studio/blob/c1c2b3e7a84bce9fcce470ca824a15aa528c1390/src/studio/src/repositories/gitea-config/app.ini#L36

I have not verified properly, but it looks a lot like this is for local dev, and the other one is for kubernetes deployments. Currently local dev is http only, so be carefull before accepting this.

---

We are merging these two files so the same file is used for both local dev and kubernetes.
For local dev we are overriding the COOKIE_SECURE setting it to false [here](https://github.com/Altinn/altinn-studio/blob/feature/6180-rootless-gitea/src/studio/docker-compose.yml#L82).
As a default COOKIE_SECURE is true, like it was in the old kubernetes app.ini

Does this resolve your concerns @ivarne ?

---

Yes, it does.